app/views/idv/index.html.slim

```diff
@@ -4,41 +4,37 @@ h1.h3.my0 = t('idv.titles.expectations')
 
 .mb-40p
   .mb-40p
-    | To complete registration with login.gov, we’ll ask you about the following:
+    | To verify your identityv, we’ll ask you to answer a few questions
   h2.mb1.pb-tiny.h5.border-bottom.border-teal Basic information
-  .mb-12p.h6 We’ll start with basic information, like:
+  .mb-12p.h6 We need to know some personal details
   ul.mb0.teal-dots
     li.mb1 Full name
-    li.mb1 Date of birth
     li.mb1 Mailing address
+    li.mb1 Date of birth
     li.mb1 Social Security Number or Individual Tax Identification Number
 
 .mb-40p
   h2.mt0.mb1.pb-tiny.h5.border-bottom.border-teal Financial information
   .mb-12p.h6
-    | Additionally, we need a number from only one of your financial accounts.
-      We can use any of the following:
+    | We need one of the following:
   ul.mb-12p.teal-dots
     li.mb1 Last 8 digits of a credit card
     li.mb1 Auto loan account number
     li.mb1 Mortgage loan account number
     li.mb1 Home equity line of credit account number
   .h6.italic
-    | You will never be charged any money, and are not sharing any account balances,
-      or other financial information with us.
+    | You are not sharing any account balances or transaction details, 
+      and we will never charge any fees.
 
 .mb-40p
-  h2.mt0.mb1.pb-tiny.h5.border-bottom.border-teal Official phone number
-  .mb-12p.h6 Finally, we need your official phone number, which can be:
+  h2.mt0.mb1.pb-tiny.h5.border-bottom.border-teal Phone line in your name
+  .mb-12p.h6 We use telephone company records to verify your identity. The phone number you give us: 
   ul.mb0.teal-dots
-    li.mb1 Lorem ipsum dolor set
-    li.mb1 Lorem ipsum dolor
-
-p.mb3
-  | We have partnered with a trusted third party company to verify this information
-    and your identity.
+    li.mb1 Be in your name, or a family member's name
+    li.mb1 Not be a virtual phone, such as Google Voice or Skype
+    li.mb1 Not be a pay-as-you-go phone
 
-p.mb3.bold Do you have all of this information available?
+p.mb3.bold Do you have all of this information available right now?
 
 = link_to 'Yes', idv_session_path, class: 'btn btn-primary btn-wide'
```

Comment:
This button copy will change when we do https://github.com/18F/identity-private/issues/1012